Pipeline/SimpleBuildReportDeploy/README.md

```diff
@@ -0,0 +1,293 @@
+# Simple Build Report Deploy (Deploy the package to target libraries)
+## Description
+This "SimpleBuildReportDeploy.groovy" sample script is capable of untaring the tar package produced by [PackageBuildOutputs](../PackageBuildOutputs/) and deploy the tar package contents to the target libraries. 
+
+Potential scenarios include:
+* Deploying the tar package to the target environment libraries.
+* If the user wants to do shift-left testing using IBM Z Virtual Test Platform, and subsystem updates are not required.
+
+As of now, this script does not perform any activation activities like a CICS NEWCOPY or a DB2 BIND.  Currently this script does not support rollback process also.
```

Comment:
Rephrase `Currently this script does not support rollback process also.` to sth like `Also, it does not support any type of rollback process.`

Reply:
Rephrased as advised and created a new pull request #148.

Reply:
Please ignore #148 and consider #149 .